package.json

```diff
@@ -119,36 +119,36 @@
   "devDependencies": {
     "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "^14.5.0",
-    "@types/gapi": "^0.0.44",
-    "@types/gapi.auth2": "^0.0.57",
+    "@types/gapi": "^0.0.45",
+    "@types/gapi.auth2": "^0.0.58",
     "@types/gapi.client.drive": "^3.0.15",
     "@types/gapi.client.people": "^1.0.5",
-    "@types/google.accounts": "^0.0.9",
-    "@types/jest": "^29.5.4",
+    "@types/google.accounts": "^0.0.10",
+    "@types/jest": "^29.5.5",
     "@types/lodash.debounce": "^4.0.7",
     "@types/luxon": "^3.3.2",
-    "@types/node": "^20.6.1",
+    "@types/node": "^20.7.0",
     "@types/pako": "^2.0.0",
-    "@types/react": "^18.2.21",
+    "@types/react": "18.2.21",
```

Comment:
We can't upgrade to 18.2.22 because of https://github.com/DefinitelyTyped/DefinitelyTyped/issues/66841